### PR TITLE
Some nullable client field docs, and other misc docs

### PR DESCRIPTION
**Pull request by i509VCB**

### CODE REVIEW

**Review by jaskarth — APPROVED** [review]

File: mappings/net/minecraft/client/MinecraftClient.mapping [hunk 1]
```diff
@@ -3,6 +3,8 @@ CLASS net/minecraft/class_310 net/minecraft/client/MinecraftClient
 	FIELD field_16762 game Lnet/minecraft/class_3799;
 	FIELD field_1686 windowProvider Lnet/minecraft/class_3682;
 	FIELD field_1687 world Lnet/minecraft/class_638;
+		COMMENT Represents the world the client is currently viewing.
+		COMMENT This field is not null when in game.
```
Comment (liach) [inline mappings/net/minecraft/client/MinecraftClient.mapping:7]: Should say "connected to a logical server" than "in game"

File: mappings/net/minecraft/world/World.mapping [hunk 2]
```diff
@@ -85,6 +85,9 @@ CLASS net/minecraft/class_1937 net/minecraft/world/World
 		ARG 2 entity
 		ARG 3 direction
 	METHOD method_27982 isDebugWorld ()Z
+		COMMENT Checks if this world is a debug world.
+		COMMENT
+		COMMENT <p>Debug worlds are not modifiable and are typically meant for development and debug use only.
```
Comment (liach) [inline mappings/net/minecraft/world/World.mapping:90]: maybe just link to minecraft wiki

File: mappings/net/minecraft/client/MinecraftClient.mapping (hunk 1 above)
Comment (shartte) [reply mappings/net/minecraft/client/MinecraftClient.mapping:7]: I think this is fine, this field in particular only says "currently, we're viewing _THIS_ world". It doesn't really say anything about a server connection in my opinion. (also: isn't this field null while connected and still in the login phase?)

File: mappings/net/minecraft/world/World.mapping (hunk 2 above)
Comment (shartte) [reply mappings/net/minecraft/world/World.mapping:90]: This would be the relevant link:
https://minecraft.gamepedia.com/Debug_mode

**Review by shartte — APPROVED** [review]

File: mappings/net/minecraft/client/MinecraftClient.mapping [hunk 3]
```diff
@@ -244,6 +248,8 @@ CLASS net/minecraft/class_310 net/minecraft/client/MinecraftClient
 	METHOD method_24460 endMonitor (ZLnet/minecraft/class_4758;)V
 		ARG 1 active
 		ARG 2 monitor
+	METHOD method_27022 shouldGlow (Lnet/minecraft/class_1297;)Z
```
Comment (YanisBft) [inline mappings/net/minecraft/client/MinecraftClient.mapping]: I already mapped that in my client pr

**Review by YanisBft — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

File: mappings/net/minecraft/world/World.mapping (hunk 2 above)
Comment (liach) [inline mappings/net/minecraft/world/World.mapping:90]:
```suggestion
		COMMENT <p>Debug worlds are not modifiable and are typically meant for development and debug use only.
		COMMENT See <a href="https://minecraft.gamepedia.com/Debug_mode">the minecraft wiki</a> as well.
```

**Review by liach — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**